Sources/Moya/MoyaProvider.swift

```diff
@@ -77,30 +82,37 @@ open class MoyaProvider<Target: TargetType> {
     /// Designated request-making method. Returns a `Cancellable` token to cancel the request later.
     @discardableResult
     open func request(_ target: Target, completion: @escaping Moya.Completion) -> Cancellable {
-        return self.request(target, queue: nil, completion: completion)
+        return self.request(target, callbackQueue: callbackQueue, completion: completion)
     }
 
-    /// Designated request-making method with queue option. Returns a `Cancellable` token to cancel the request later.
+    /// Designated request-making method with an optional callback queue. Returns a `Cancellable` token to cancel the request later.
     @discardableResult
-    open func request(_ target: Target, queue: DispatchQueue?, progress: Moya.ProgressBlock? = nil, completion: @escaping Moya.Completion) -> Cancellable {
-        return requestNormal(target, queue: queue, progress: progress, completion: completion)
+    open func request(_ target: Target, callbackQueue: DispatchQueue?, progress: Moya.ProgressBlock? = nil, completion: @escaping Moya.Completion) -> Cancellable {
```

Comment:
@AndrewSB I had similar thoughts. It defaults to self inside the method body with a nil coalescing operator. 
```Swift
let callbackQueue = callbackQueue ?? self.callbackQueue
```
We could possibly explicitly mark this parameter as `nil` by default?